content/docs/visualization/consoles.md

```diff
@@ -9,13 +9,16 @@ The Console templates allow for creation of arbitrary consoles using the [Go
 templating langauge](http://golang.org/pkg/text/template/). These are served
 from the Prometheus server.
 
-Console templates are the most powerful way to create templates that can be easily managed in source control, there is a learning curve though so 
-users new to this style of monitoring should try out [PromDash](../promdash/) first.
+Console templates are the most powerful way to create templates that can be
+easily managed in source control, there is a learning curve though so users new
+to this style of monitoring should try out [PromDash](../promdash/) first.
 
 ## Getting started
 
-Prometheus comes with an example set of consoles to get you going, these can be found at `/consoles/index.html.example` on a running Prometheus
-and will let you see Node Exporter consoles if your node exporters have a `job="node"` label.
+Prometheus comes with an example set of consoles to get you going, these can be
+found at `/consoles/index.html.example` on a running Prometheus and will
+display Node Exporter consoles if Prometheus is scrapingnode exporters with a
+`job="node"` label.
 
 Consoles have 5 parts:
```

Comment:
This sounds like you are talking about consoles in general, while you are talking about the example templates.

Say "The example consoles have five parts:"